### PR TITLE
Update mermaid for 1.21

### LINKED ISSUE #28170: [1.21.0] Mermaid on Wiki no longer working
### Description

Hello Everyone,
just updated from `1.20.5` to `1.21.0` and the `mermaid` git diagram on a Wiki page breaks with the following error `Qa(...).catch is not a function` and nothing in JS console, just the plain source text of the diagram below the error.

Same error if starting from a fresh `1.21.0` installation.

The error cannot be reproduced on the _Gitea demo site_, `1.22.0+dev-350-g2d9b91aa2`.

https://blog.gitea.com/release-of-1.21.0/ doesn't provide any hint/advice about Mermaid diagrams.

Thanks in advance,
Matteo

### Gitea Version

1.21.0

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

Docker Rootless Image

### How are you running Gitea?

Using the _Docker rootless image_, `docker.io/gitea/gitea:1.21.0-rootless`, by means of a quite standard compose.

### Database

MySQL/MariaDB

### PR DESCRIPTION
**Pull request by wxiaoguang**
Try to fix #28170

### CODE REVIEW

**Review by delvh — APPROVED** [review]

**Review by lunny — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -7530,9 +7530,9 @@
       }
     },
     "node_modules/mermaid": {
-      "version": "10.5.0",
-      "resolved": "https://registry.npmjs.org/mermaid/-/mermaid-10.5.0.tgz",
-      "integrity": "sha512-9l0o1uUod78D3/FVYPGSsgV+Z0tSnzLBDiC9rVzvelPxuO80HbN1oDr9ofpPETQy9XpypPQa26fr09VzEPfvWA==",
+      "version": "10.6.1",
+      "resolved": "https://registry.npmmirror.com/mermaid/-/mermaid-10.6.1.tgz",
```
Comment (silverwind) [inline package-lock.json:7534]: @wxiaoguang Don't think it causes any issues but we should avoid writing non-official npm mirrors into the lockfile.

Comment (wxiaoguang) [reply package-lock.json:7534]: Agree, maybe it's because at that time I was using npmmirror. Will pay more attention

Comment (silverwind) [reply package-lock.json:7534]: If it becomes a problem, a simple find-replace should be able to fix it. I don't think the npm client actually uses this URL, but other clients might.

Comment (silverwind) [reply package-lock.json:7534]: https://github.com/go-gitea/gitea/pull/30719 adds a linter for it.

**Pull request merged**